config/webpack.config.base.js

```diff
@@ -62,7 +63,7 @@ const MEDIA_URL_LOADER = {
 };
 
 const CSS_LOADER = {
-  loader: 'style!css',
+  loader: ExtractTextPlugin.extract('css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]'),
```

Comment:
oh huh, I was going off a tutorial for how to set this up. It works as is, but I can try to change it if you want